### PR TITLE
feat: add card component

### PR DESCRIPTION
**Pull request by adith-a-danthi**
Added card component
- Elevated, outlined card.
- Cards with Images
- Cards with Text Overlay
- E-commerce card

image

### CODE REVIEW

File: docs/components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+.card-landscape {
+  display: flex;
+}
+
+.card-img {
+  width: 100%;
+  height: auto;
+  background-repeat: no-repeat;
+  object-fit: cover;
```
Comment (ArjunGTX) [inline docs/components/card/card.css:28]: suggestion: use `object-position: center` to make the image more responsive

File: docs/components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  color: var(--color-gray);
+}
+.card-badge.badge-gray {
+  background-color: var(--color-gray);
+}
+
+.card-dismiss {
+  position: absolute;
+  top: 0.5rem;
+  right: 0.5rem;
```
Comment (ArjunGTX) [inline docs/components/card/card.css:71]: Better to store these values in variables since they are used repeatedly

**Review by ArjunGTX — APPROVED** [review]

**Comment by ArjunGTX** [comment]
Everything else looks good

File: docs/components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
+.card {
+  position: relative;
+  min-width: 0;
+  font-weight: 400;
+  background-color: white;
+  box-shadow: 0 1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1);
```
Comment (Hamdrive) [inline docs/components/card/card.css:6]: Prefer using CSS variables instead

**Review by Hamdrive — COMMENTED** [review]
Good work done
Suggestions given, ready to push!

**Pull request merged**